### ISSUE TITLE
Groups

### ISSUE THREAD

**Issue by kwinkunks** [agile-geoscience/pickr#192]
To facilitate use in the classroom, Brian Romans suggested having a way to filter results based on some specified group of users. See [this Twitter thread](https://twitter.com/clasticdetritus/status/587675173844754432).

I think this goes nicely alongside other results filters, e.g. 

- only see picks with *x* upvotes or more
- only see picks by people I follow
- only see picks by people in the same email domain as me (?)
- only see picks by people with reputation > *y*
- only see picks by people in group *z* (this ticket)

If we can find a frictionless way to form groups, so much the better. Maybe the answer is just to have arbitrary user-defined lists, like in Twitter and Google Plus. That way Brian could make the group, and not rely on all the students adding themselves. Then we'd just need a way to ease discovery. 

As for generating the results, I guess we'd make these heatmaps on the fly? Probably all much easier once we have a workable JS solution for the rendering. I don't like the thought of using `pickthis.py` for this, and we can't pre-render all the permutation.

**Comment by stevejpurves** [comment]
yes. definitely need to move the heatmap generation to the frontend for
this one.

On 13 April 2015 at 19:02, Matt Hall <notifications@github.com> wrote:

> To facilitate use in the classroom, Brian Romans suggested having a way to
> filter results based on some specified group of users. See this Twitter
> thread <https://twitter.com/clasticdetritus/status/587675173844754432>.
>
> I think this goes nicely alongside other results filters, e.g.
>
>    - only see picks with *x* upvotes or more
>    - only see picks by people I follow
>    - only see picks by people in the same email domain as me (?)
>    - only see picks by people with reputation > *y*
>    - only see picks by people in group *z* (this ticket)
>
> If we can find a frictionless way to form groups, so much the better.
> Maybe the answer is just to have arbitrary user-defined lists, like in
> Twitter and Google Plus. That way Brian could make the group, and not rely
> on all the students adding themselves. Then we'd just need a way to ease
> discovery.
>
> As for generating the results, I guess we'd make these heatmaps on the
> fly? Probably all much easier once we have a workable JS solution for the
> rendering. I don't like the thought of using pickthis.py for this, and we
> can't pre-render all the permutation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/agile-geoscience/pickr/issues/192>.
>



-- 
españa: 622 195193
uk: 020 32 893 858
@stevejpurves
airpair{}me/stevejpurves
<http://t.signauxhuit.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XYg7fcFmHN8pTrfRdnPr-W3Mx_5x56dJcRf5fm60W02?t=http%3A%2F%2Fairpair.me%2Fstevejpurves%3Futm_source%3Dexpert%26utm_medium%3Demail%26utm_content%3Dsignature%26utm_campaign%3Dairpairme&si=6323770510278656&pi=768607e9-ddd7-42cd-a553-efb06f71d81c>
skype: steve.purves.home
http://www.linkedin.com/in/stevejpurves

**Comment by kwinkunks** [comment]
More on this in an email from Brian...

>>> what would be great is to be able to have my
students (16-20) each be able to pick and then I can display the heat
map of all their picks together. I can envision having a great
discussion with them about the various interpretations. I think this
would significantly enhance their learning. And then, when we are
finished, their picks can be added to the open/community picks and they
can see how it compares to that much larger sample set. This would no
doubt initiate further discussion.
